### PR TITLE
Operations Branch

### PR DESCRIPTION
**Pull request by sdmnc**

### CODE REVIEW

File: firstDirectory/topic3/Operations.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+public class Operations {
+    static void printArithmeticOperations() {
+        System.out.println("ArithmeticOperations");
+        int a = 10;
+        int b = 110;
+        int sum = a+b;
+        int min = b-a;
+        int der = a*b;
+        double quotient = b/a;
```
Comment (dmitryturcan) [inline firstDirectory/topic3/Operations.java:17]: please consider using the % operator (restul impartirii) with int number

**Review by dmitryturcan — CHANGES_REQUESTED** [review]
please add .idea folder to .gitignore file, this is the local env config project which is unique per each computer and user, no need to store it in git

**Comment by sdmnc** [comment]
> please add .idea folder to .gitignore file, this is the local env config project which is unique per each computer and user, no need to store it in git
Thank you for review. I hope I did it corectly.

File: firstDirectory/topic3/Operations.java (hunk 1 above)
Comment (sdmnc) [reply firstDirectory/topic3/Operations.java:17]: > please consider using the % operator (restul impartirii) with int number

Thank you! Added b%a arithmeticl operation.

**Pull request closed without merging**